examples/kubernetes/kafka-oauth-single-spring.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: kafka.strimzi.io/v1beta1
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    version: 2.4.1
```

Comment:
Should this have 2.5.0 as well?

Reply:
I like to use different versions to increase the probability of catching compatibility problems that we may have missed. This should work with versions 2.4.0, 2.4.1, and 2.5.0, but not 2.3.0 because the latest image for that would not contain the latest operator any more.
The whole decision which of the supported versions to use in this place is rather non-deterministic :)
One benefit of using 2.5.0 might be to prod users to use the latest. OTOH, using older version makes it clear that you don't have to use the latest if for some reason you don't want to or you can't.

Reply:
Ok - If that was intentional to keep it as 2.4.1 I'm fine with it. It was not really clear from the PR. Thanks.